app/ts/components/requirements.tsx

```diff
@@ -123,11 +145,11 @@ export const Immutable = ( { checkBoxesArray } : { checkBoxesArray: OptionalSign
 		<div style = 'padding-top: 30px; padding-bottom: 30px; align-items: center; display: grid; width: 100%'>
 			{ checkBoxes.immutable ? <>
 				<p class = 'status-green'>
-					{`IMMUTABLE for about ${ dateDiff } (until ${ dateString })` }
+					{`Almost IMMUTABLE* for about ${ dateDiff } (until ${ dateString })` }
 				</p>
 			</>: <p class = 'status-red'> NOT IMMUTABLE </p> }
 		</div>
-		{ checkBoxes.immutable ? <></> : <p style = 'margin: 0px; margin-bottom: 10px; padding-left: 10px;' class = 'requirement'> { checkBoxes.domainInfo.subDomain } should satisfy the following conditions to be immutable: </p> }
+		{ checkBoxes.immutable ? <></> : <p style = 'margin: 0px; margin-bottom: 10px; padding-left: 10px;' class = 'requirement'> { checkBoxes.domainInfo.subDomain } should satisfy the following conditions to be almost immutable*: </p> }
```

Comment:
```suggestion
		{ checkBoxes.immutable ? <></> : <p style = 'margin: 0px; margin-bottom: 10px; padding-left: 10px;' class = 'requirement'> { checkBoxes.domainInfo.subDomain } should satisfy the following conditions to be immutable: </p> }
```
The list of conditions includes the checkbox for "convince ENS DAO to fix the bugs", so technically if someone can complete all of the conditions it would be fully immutable, so no need for asterisk I think.